packages/framework/assets/js/admin/components/CkEditorPreview.js

```diff
@@ -0,0 +1,45 @@
+import Register from '../../common/utils/Register';
+import Translator from 'bazinga-translator';
+
+export default class CkEditorPreview {
+
+    constructor ($ckEditorPreview) {
+        const expandText = Translator.trans('... expand ...');
+        const collapseText = Translator.trans('collapse');
```

Comment:
This is never displayed

Reply:
Actually, it is. 
The longer text is collapsed by default and can be expanded with '...expand...' button. When text is expanded (ckeditor is not initialized), this text is rendered at the bottom of the expanded field and allows to collapse the long text back.

Reply:
Reworked, so now only edit button is present (expand preview text functionality was removed)